src/App.js

```diff
@@ -17,8 +18,19 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+class App extends React.Component {
+  state = {
```

Comment:
Why do you need an empty state here?

Reply:
Agree) You need state for keeping selected tab content and rendering it in this component